src/formElementPlugins/ApplicationViewWrapperNEW.tsx

```diff
@@ -149,15 +149,72 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperPropsNEW> = (props)
     />
   )
 
+  const changesToResponseProps: ChangesToResponseWarningProps = {
+    currentReview,
+    isChanged,
+    isValid: validationState ? (validationState.isValid as boolean) : true,
+  }
+
+  const displayResponseWarning = !!currentReview || (isChanged && validationState.isValid)
+
   return (
     <ErrorBoundary pluginCode={pluginCode}>
       <React.Suspense fallback="Loading Plugin">
-        {parametersReady && <Form.Field required={isRequired}>{PluginComponent}</Form.Field>}
+        <div
+          style={{
+            border: displayResponseWarning ? 'solid 1px' : 'transparent',
+            borderColor: isChanged ? 'blue' : !!currentReview ? 'red' : 'black',
+            padding: '5px',
+            margin: '5px',
+          }}
+        >
+          {parametersReady && <Form.Field required={isRequired}>{PluginComponent}</Form.Field>}
+          <ChangesToResponseWarning {...changesToResponseProps} />
+        </div>
       </React.Suspense>
     </ErrorBoundary>
   )
 }
 
+interface ChangesToResponseWarningProps {
+  currentReview?: ReviewResponse
+  isChanged: boolean
+  isValid: boolean
+}
+
+const ChangesToResponseWarning: React.FC<ChangesToResponseWarningProps> = ({
+  currentReview,
+  isChanged,
+  isValid,
+}) => {
+  const displayResponseWarning = currentReview || (isChanged && isValid)
+
+  return (
+    <div
+      style={{
+        color: isChanged ? 'grey' : 'red',
+        minHeight: 18, // Used to keep space for warning when not displaying
+      }}
+    >
+      <Icon
+        style={{
+          display: displayResponseWarning ? 'inline-block' : 'none',
+        }}
+        name={
+          currentReview
+            ? isChanged
+              ? 'comment alternate outline'
```

Comment:
Don't love this icon, doesn't seem to convey anything. How does no icon look for this. The other two are error and info icons, which make sense, but once it's correct, there's nothing really to inform the user of, so maybe doesn't need any? (You've still got the border to show it's changed)

Reply:
I'm not sure which one you're talking abut? This one: 
<img width="354" alt="Screen Shot 2021-03-19 at 5 27 34 PM" src="https://user-images.githubusercontent.com/16461988/111731489-74462680-88d8-11eb-8dc3-c1f91efa3d9e.png">

In this case - that's what are in the designs. I was just following orders : P

Reply:
Oh, really? Okay. Save it for another day discussion then :)